docs/source/release/v5.1.0/mantidworkbench.rst

```diff
@@ -83,6 +83,8 @@ Improvements
 - Added an option in the settings to specify the default legend size.
 - Added an option to the settings window to set the default colormap for image plots.
 - Improved loading of python plugins at startup on slow disks.
+- Sliceviewer no longer lists the reversed colourmaps along with the regular, instead they are accessed with a reverse checkbox.
+- Sliceviewer colourmap uses the default colourmap from the settings.
```

Comment:
Was this added in a previous PR but not added to the release notes at the time?

Reply:
The PR that put in the default colourbar says it for image plots (line 84) I wanted to clarify it was for slice viewer as well.